src/scancode/cli.py

```diff
@@ -231,11 +231,51 @@ class ScanCommand(BaseCommand):
     short_usage_help = '''
 Try 'scancode --help' for help on options and arguments.'''
 
-    def get_params(self, ctx):
+    def format_options(self, ctx, formatter):
```

Comment:
Just to make sure I get this: is this overriden from the click class?
If so, a mention in the docstring would be nice for future understaning